components/dot_menu/dot_menu.test.jsx

```diff
@@ -1,11 +1,9 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {shallow, ShallowWrapper} from 'enzyme';
-
 import React from 'react';
 
-import {PostType} from 'mattermost-redux/types/posts';
+import {shallowWithIntl} from 'tests/helpers/intl-test-helper';
```

Comment:
This file got converted from typescript to javascript

Reply:
All these files back to Javascript were done like this because Typescript was giving tons of problems with the components with the intl injected. Feel free to give it another try, but the simplest solution was going back to javascript for the tests.